src/sota_tools/request_pool.h

```diff
@@ -42,7 +42,7 @@ class RequestPool {
   RateController rate_controller_;
   int running_requests_;
   int total_requests_made_{0};
-  const TreehubServer& server_;
+  TreehubServer& server_;
```

Comment:
It's too bad we lose this const-ness. It doesn't seem so critical, though, and I don't have a better solution available yet, though, so maybe it's fine.

Reply:
Yes, I feel sorry for the loss too. I have tried.